### PR TITLE
CSPL-1765: fixing a log format that is causing the panic

### PR DESCRIPTION
**Pull request by sgontla**
fixing a log format that is causing the panic

### CODE REVIEW

**Review by jryb — APPROVED** [review]

File: pkg/splunk/util/secrets.go [hunk 1]
```diff
@@ -482,8 +482,7 @@ func ApplyNamespaceScopedSecretObject(ctx context.Context, client splcommon.Cont
 	}
 
 	// Make data
-	scopedLog.Info("Namespace scoped secret does not exist, creating it ",
-		"and filling it with new values for all token types")
+	scopedLog.Info("Namespace scoped secret does not exist, creating and filling it with new values for all token types")
```
Comment (akondur) [inline pkg/splunk/util/secrets.go:485]: Is this format causing a panic? Are there logs for the panic?

Comment (sgontla) [reply pkg/splunk/util/secrets.go:485]: @akondur , right, this is causing the panic with the new SDK (and I guess mostly with the new lib packages). As the interface implementation expects the odd number of arguments (msg[, KV pairs]) format, it is doing a panic. In older releases, it was throwing a warning and continuing. 

I think, as a permanent fix, we need to add a format checking logic(something like `make log-fmt`), that validates all the log entries and finds out any deviations from:
1. error: even number of arguments
2. info: odd number of entries

Comment (sgontla) [reply pkg/splunk/util/secrets.go:485]: @jambrosiano, can you share a panic log example here.

**Review by akondur — APPROVED** [review]

File: pkg/splunk/util/secrets.go (hunk 1 above)
Comment (akondur) [reply pkg/splunk/util/secrets.go:485]: To add to this maybe we should monitor operator logs during tests for any PANIC's? @pdhanoyasplunk @jambrosiano @tpereirasplunk wonder if this is possible.

**Review by smohan-splunk — APPROVED** [review]

**Pull request merged**